.github/workflows/ci.yaml

```diff
@@ -7,17 +7,18 @@ on:
     paths-ignore:
       - ".github/workflows/prebuild.yaml"
 
-jobs: 
+jobs:
   Linux:
     name: Test on Linux
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 12, 14, 16, 18]
+        node: [14, 16, 18]
```

Comment:
We are still supporting these versions so I don't think we should remove them until the next major version when we can bump our minimum Node.js version...

Reply:
I see, should I open a separate PR to drop support for them? Is there anything that should land before we can bump the minimum Node.js version?